src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -8,17 +8,32 @@
 from azure.cli.core.azclierror import InvalidArgumentValueError, RequiredArgumentMissingError
 from knack.log import get_logger
 
-from ..vendored_sdks.models import ExtensionInstance
-from ..vendored_sdks.models import ExtensionInstanceUpdate
-from ..vendored_sdks.models import ScopeCluster
-from ..vendored_sdks.models import Scope
+from azure.cli.core.commands.client_factory import get_subscription_id

```

Comment:
Can you please localize common imports i.e. put `from azure.cli.core...` imports together